.checkpatch.conf

```diff
@@ -0,0 +1,19 @@
+--no-tree
+--strict
+--ignore=SPLIT_STRING
+--ignore SSCANF_TO_KSTRTO
+--ignore=NEW_TYPEDEFS
+--ignore=DEPRECATED_VARIABLE
+--ignore=COMPARISON_TO_NULL
+--ignore=BIT_MACRO
+--ignore=PREFER_PRINTF
+--ignore=PREFER_SCANF
+--ignore=VOLATILE
+--ignore=AVOID_EXTERNS
+--ignore=CONST_STRUCT
+--ignore=PREFER_KERNEL_TYPES
+--ignore=CONSTANT_COMPARISON
+--ignore=BLOCK_COMMENT_STYLE
+--ignore=UNNECESSARY_PARENTHESES
```

Comment:
These flags are different from the ones in `scripts/checkpatch_inc.sh`. It seems to me that some consolidation is needed, i.e., choose which one to keep and have all the flags here rather than in the `.sh` file.

Reply:
+1